From 212e0f8ffeea470f17f15cee59f3764b3d5e3f0b Mon Sep 17 00:00:00 2001
From: Kent Russell <kent.russell@amd.com>
Date: Wed, 18 Jan 2017 12:47:43 -0500
Subject: [PATCH 5448/5855] Revert "drm/amdkfd: destroy mutex if process
 creation fails"

This reverts commit 7fd5e03ca6b41a591bd9fda083362b8a07cfb5f7.
---
 drivers/gpu/drm/amd/amdkfd/kfd_process.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_process.c b/drivers/gpu/drm/amd/amdkfd/kfd_process.c
index a069c3d..39332e6 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_process.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_process.c
@@ -618,7 +618,6 @@ static struct kfd_process *create_process(const struct task_struct *thread)
 	synchronize_rcu();
 	mmu_notifier_unregister_no_release(&process->mmu_notifier, process->mm);
 err_mmu_notifier:
-	mutex_destroy(&process->mutex);
 	kfd_pasid_free(process->pasid);
 err_alloc_pasid:
 	kfree(process->queues);
-- 
2.7.4

